src/spring-cloud/azext_spring_cloud/_validators_enterprise.py

```diff
@@ -15,6 +17,8 @@
     is_enterprise_tier
 )
 
+BUILDPACKS_BINDING_NAME_REGEX_PATTTERN=r"(^[a-zA-Z]$|^[a-zA-Z][-a-zA-Z0-9]*[a-zA-Z0-9]$)"
+BUILDPACKS_BINDING_NAME_MAX_LENGTH=19
```

Comment:
suggest to put the description of why 19 here as we discussed offline.

Reply:
OK